llama_stack/providers/tests/agents/fixtures.py

```diff
@@ -55,7 +61,28 @@ def agents_meta_reference() -> ProviderFixture:
     )
 
 
+@pytest.fixture(scope="session")
+def tool_runtime_memory() -> ProviderFixture:
```

Comment:
why is this fixture in tests/agents and not in tests/tool_runtime?

Reply:
added new tools tests and fixtures.